scipy/linalg/matfuncs.py

```diff
@@ -668,3 +675,329 @@ def rounded_sign(x):
         return S0
     else:
         return S0, errest
+
+
+def is_matrix_hermitian(A):
```

Comment:
`is_hermitian()`

Reply:
It's been improved with tolerance and a calculation based on norm.